[DOCUMENTATION] Update plugins documentation (#1860)
kitodo/kitodo-presentation: Documentation/Plugins/Index.rst

Collection
----------

The collection plugin shows one collection, all collections or selected collections. It can be used to display the list

> of collections or the overview of single collection. The plugin is configured in the TypoScript and in the FlexForm.

Reviewer: ```suggestion
of collections or the contents of a single collection. The plugin is configured in the TypoScript and in the FlexForm.
```

   :Default:
      By default all features are activated. The selection is stored as comma separated list.

>        doublePage,pageFirst,pageBack,pageStepBack,pageSelect,pageForward,pageStepForward,pageLast,listView,zoom,rotation,measureForward,measureBack

Reviewer: ```suggestion
```
Is this duplication intentional?

Author: Duplication? It is correction from '_litView_' to '_listView_'.

Reviewer: Sorry, that was misleading! I was refering to the following lines where all options are listed again. So, currently there are all options as comma-separated list and then immediately afterwards all options again as a pointed list. IMHO the pointed list under `Values:` alone would be sufficient.

Author: Actually not all options are here, currently two more options are possible: 
* pagesBackward
* pagesForward

They replace single forward and backward navigations.

 - :Property:
       tools
   :Data Type:
       `t3tsref:data-type-string`
   :Default:
   :Values:
       * annotationtool
       * audiovideotool
       * fulltextdownloadtool
       * fulltexttool
       * imagedownloadtool
       * imagemanipulationtool
       * modeldownloadtool
       * multiviewaddsourcetool
       * pdfdownloadtool
       * searchindocumenttool
       * scoretool
       * tx_dlf_annotationtool
       * tx_dlf_audiovideotool
       * tx_dlf_fulltextdownloadtool
       * tx_dlf_fulltexttool
       * tx_dlf_imagedownloadtool
       * tx_dlf_imagemanipulationtool
       * tx_dlf_modeldownloadtool
       * tx_dlf_multiviewaddsourcetool
       * tx_dlf_pdfdownloadtool
       * tx_dlf_searchindocumenttool

>        * tx_dlf_scoretool

Reviewer: Shouldn't we only propose one variant of each tool here? (I'd prefer the option without `tx_dlf_` prefix.)

Author: We can do it so, I would also prefer only variant without the prefix.

>        * tx_dlf_scoretool

Reviewer: ```suggestion
       * annotationtool
       * audiovideotool
       * fulltextdownloadtool
       * fulltexttool
       * imagedownloadtool
       * imagemanipulationtool
       * modeldownloadtool
       * multiviewaddsourcetool
       * pdfdownloadtool
       * searchindocumenttool
       * scoretool
```
